pkg/api/controller.go

```diff
@@ -3829,7 +3829,7 @@ func (c *Controller) GetSetupState(w http.ResponseWriter, r *http.Request) {
 
 	state := string(savedState)
 	// no need to create an admin user if users are managed externally
-	if c.Config.IsAuthTypeAPI() {
+	if c.Config.Auth.UIConfig.RBAC == config.AuthRBACExternal {
```

Comment:
I prefer the func version: it will let us add additional conditions if we need to add them.  For instance, suppose we add a _third_ type of "RBAC" configuration.  One example: we might want to support "some groups simplified but others not".  Then these `if` statements become a liability.

Reply:
In this is case we wanted to make sure we match to one of the state that we do not manage uses.
Relay only on endpoint existence is no longer the case